apps/teams-test-app/src/components/utils/ApiWithoutInput.tsx

```diff
@@ -18,7 +24,17 @@ export const ApiWithoutInput = (props: ApiWithoutInputProps): React.ReactElement
         name={`button_${name}`}
         type="button"
         value={title}
-        onClick={async () => setResult(await onClick(setResult))}
+        onClick={async () => {
+          if (typeof onClick == 'function') {
```

Comment:
Can I know if you think a triple equality with '===' is better here?
` if (typeof onClick === 'function')`

Reply:
This was just a typo, good catch! I'll update it